### PR TITLE
[Improvement] Improve language key name and fix typo

### LINKED ISSUE #3251: [Improvement]: Typos in en.ts and zh.ts
### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/amoro/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

- Use camel case naming
	- addgroup -> addGroup
	- ...
- Typo `partiton` -> `partition`
- Typo `welecomeTip` -> `welcomeTip`
- Duplicate `optimizergroup`
...

### How should we improve?

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Subtasks

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by chouchouji**
## Why are the changes needed?

Close #3251 .

## Brief change log

-

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by chouchouji** [comment]
@zhoujinsong 

These three variables are same? If they are, I will remove two variables.

image

File: amoro-web/src/language/en.ts [hunk 1]
```diff
@@ -127,8 +127,8 @@ export default {
   resourceGroup: 'Resource Group',
   releaseOptModalTitle: 'Release this optimizer？',
   cancelOptimizingProcessOptModalTitle: 'Cancel the optimizing process of this table？',
```
Comment (zhongqishang) [inline amoro-web/src/language/en.ts:129]: There are four places where ? is a full-width character.

**Comment by zhongqishang** [comment]
> These three variables are same? If they are, I will remove two variables.

I found four place and I suggest keeping the last two, they have singular and plural forms.
```
  optimzerGroup: 'Optimizer Group',
  optimizergroup: 'Optimizer Groups',
  optimizerGroup: 'Optimizer Group', --keeping
  optimizerGroups: 'Optimizer Groups',, --keeping
```

**Comment by zhongqishang** [comment]
And recommended to leave a space before and after the number.
```
  last1h: '最近 1小时',
  last12h: '最近 12小时',
  last24h: '最近 24小时',
  last7day: '最近 7天',
```

**Comment by zhoujinsong** [comment]
> These three variables are same? If they are, I will remove two variables.

I agree with @zhongqishang.
We should keep the following variables only:
```
optimizerGroup: 'Optimizer Group',
optimizerGroups: 'Optimizer Groups',
```

File: amoro-web/src/language/en.ts (hunk 1 above)
Comment (zhoujinsong) [reply amoro-web/src/language/en.ts:129]: Yes, we should replace the question mark in Chinese(？) with the English one(?).

**Comment by chouchouji** [comment]
@zhongqishang @zhoujinsong I had updated code. Please review. Thanks!

**Review by zhongqishang — APPROVED** [review]
LGTM.

**Comment by zhoujinsong** [comment]
Thanks for the work! @chouchouji 
Thanks for the review! @zhongqishang

**Pull request merged**